.github/workflows/build-and-run-gpu.yml

```
@@ -0,0 +1,124 @@
+name: Build-And-Run Tests on GPU
+
+on:
+  pull_request:
+    # Trigger the workflow whenever a PR receives a special label
+    # in addition to the default triggering events.
+    types: [ labeled, opened, synchronize, reopened ]
+
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_and_test_gpu:
+    # Trigger on pushes to CI updates branch, or PRs with specific label
```

This comment is out-of-date now

```suggestion
    # Trigger on PRs with specific label
```